tests/integration/routes/institutions/dashboard/-components/institutional-users-list/component-test.ts

```diff
@@ -16,15 +16,27 @@ module('Integration | routes | institutions | dashboard | -components | institut
     });
 
     test('it renders and paginates', async function(assert) {
-        server.create('institution', { id: 'testinstitution' }, 'withInstitutionalUsers', 'withStatSummary');
-
+        server.create('institution', {
+            id: 'testinstitution',
+        }, 'withInstitutionUsers', 'withSummaryMetrics', 'withInstitutionDepartments');
+        const institution = await this.get('store').findRecord('institution', 'testinstitution');
+        const departmentMetrics = await institution.get('departmentMetrics');
+        const { userMetrics } = institution;
         const model = {
-            taskInstance: this.get('store').findRecord('institution', 'testinstitution'),
+            taskInstance: {
```

Comment:
Take a look at this and see if taskInstance can be updated to mock task behavior better